docs/userGuide/syntax/dates.mbdf

```diff
@@ -0,0 +1,105 @@
+## Dates
+
+{% macro njcode(raw) %}<code>{<a/>{ {{ raw }} }}</code>{% endmacro %}
```

Comment:
Not a change request, just an enquiry: did you came up with this? This syntax seems to be pretty useful for our other parts of the documentation.

Reply:
Yup